numpy/core/tests/examples/cython/checks.pyx

```diff
@@ -30,3 +30,46 @@ def get_dt64_unit(obj):
 
 def is_integer(obj):
     return isinstance(obj, (cnp.integer, int))
+
+
+def get_datetime_iso_8601_strlen():
+    return cnp.get_datetime_iso_8601_strlen(0, cnp.NPY_FR_ns)
+
+
+def convert_datetime_to_datetimestruct():
+    cdef:
+        cnp.npy_datetimestruct dts
+        cnp.PyArray_DatetimeMetaData *meta = NULL
```

Comment:
I don't think you can pass NULL.

If it doesn't work locally, it sounds a bit like you are just picking up the old `__init__.pyd` while compiling `checks.pyx`?  Maybe try using `python runtests.py` just to see if it works?

Reply:
removed the NULL.

using python runtests.py im now getting segfaults locally.  that's progress sorta.

Reply:
that helped!  im also seeing compiler warnings about make_iso_8601_datetime (in checks.pyx)  having result and outlen unitialized.